lavalink/src/node.rs

```diff
@@ -600,20 +603,21 @@ impl Drop for Connection {
 }
 
 fn connect_request(state: &NodeConfig) -> Result<Request<()>, NodeError> {
-    let mut builder = Request::get(format!("ws://{}", state.address));
-    builder = builder.header("Authorization", &state.authorization);
-    builder = builder.header("Num-Shards", state.shard_count);
-    builder = builder.header("Sec-WebSocket-Key", generate_key());
```

Comment:
should this line have been removed?

Reply:
`IntoClientRequest` actually generates the key for you, so yes, this was both intentional and correct.